frontend/src/components/episode/components/timeline-item/timeline-item.tsx

```diff
@@ -0,0 +1,52 @@
+import { ImageWrapper } from 'components/common/common';
+import { Comment as TComment } from 'common/types/types';
+import styles from './styles.module.scss';
+
+type Props = {
+  comment: TComment;
+  dimensions: {
+    offset: number;
+    progressBarWidth: number;
+    playerContainerWidth: number;
+  };
+  duration: number;
+  handleGoToTimestamp: () => void;
+};
+
+const TimelineItem: React.FC<Props> = ({
+  comment,
+  dimensions,
+  duration,
+  handleGoToTimestamp,
+}) => {
+  const commentOffset =
+    ((dimensions.offset +
+      (comment.timestamp / duration) * dimensions.progressBarWidth) /
+      dimensions.playerContainerWidth) *
+    100;
```

Comment:
move this calculation to the function `frontend/src/components/episode/components/timeline-item/helpers`